### DISCUSSION TITLE
Should the `compiler` target property really be available on all targets?

### DISCUSSION THREAD

**Discussion by oowekyala** [lf-lang/lingua-franca#783] [General]
Among the few target properties available on all targets, `compiler` strikes me as an oddity.
I can understand that this is useful in C and C++, which have several competing compiler implementations. However, AFAIK, TypeScript and Rust have a single compiler implementation, and I don't think anything useful can be put in the `compiler` property for those targets.
It's the same in Python, I think Python installs modules with pip, but there is no use case for changing the executable, right?

**Comment by lhstrh** [comment]
I don't think it should. In fact, I'm surprised that it is. We don't need it for TypeScript, Rust, and Python, I think.
